__tests__/api.test.js

```diff
@@ -32,13 +32,51 @@ describe.only("GET /api/topics", () => {
   test("404: responds with an error message saying that the requested resource does not exist", () => {
     return request(app).get("/api/topicss").expect(404);
   });
+});
+describe("GET /api", () => {
   test("200: responds with an an object of available endpoints", () => {
     return request(app)
       .get("/api")
       .expect(200)
       .then((response) => {
+        // console.log(response.body);
         const { body } = response;
         expect(body).toEqual(endpointData);
       });
   });
 });
+describe("GET /api/articles/:article_id", () => {
+  test("200: responds with an article object that contains nessesary properties.", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((response) => {
+        const { body } = response;
+        // console.log(body);
+        expect(body).toHaveProperty("article_id", expect.any(Number));
+        expect(body).toHaveProperty("title", expect.any(String));
+        expect(body).toHaveProperty("topic", expect.any(String));
+        expect(body).toHaveProperty("author", expect.any(String));
+        expect(body).toHaveProperty("body", expect.any(String));
+        expect(body).toHaveProperty("created_at", expect.any(String));
+        expect(body).toHaveProperty("votes", expect.any(Number));
+        expect(body).toHaveProperty("article_img_url", expect.any(String));
```

Comment:
While you have tested for all the keys, this test would pass if I were served article 13 when I asked for article 1. At the very least I would test that the article id is 1 as that is what you've requested in the GET.